src/scenes/Projects/Members/List/ProjectMembersList.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Breadcrumbs as BreadCrumbs,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { FC } from 'react';
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import { Breadcrumb } from '../../../../components/Breadcrumb';
+import { CreateIconButton } from '../../../../components/CreateIconButton';
+import { ProjectMemberCard } from '../../../../components/ProjectMemberCard';
+import { listOrPlaceholders } from '../../../../util';
+import { useProjectMembersQuery } from './ProjectMembers.generated';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+  },
+  breadcrumbs: {
+    marginBottom: spacing(4),
+  },
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: spacing(4),
+    height: '50px',
```

Comment:
Why is this height defined?
```suggestion
```

Reply:
The button appears after the data comes in and causes the container to expand and push the content down. I added this in to avoid that momentary resizing.